templates/stamp/run-customizations.sh

```diff
@@ -702,7 +702,17 @@ storageAccountEndpointSuffix=`get_azure_storage_endpoint_suffix ${encoded_azure_
 storage_connection_string=`generate_azure_storage_connection_string "${AZURE_ACCOUNT_NAME}" "${AZURE_ACCOUNT_KEY}" "${storageAccountEndpointSuffix}"`
 
 # create storage container for edxapp:migrate & other reporting features (containers for the database backup will be created dynamically)
-powershell -file $INSTALLER_BASEPATH/Create-StorageContainer.ps1 -AadWebClientId $AAD_WEBCLIENT_ID -AadWebClientAppKey $AAD_WEBCLIENT_APPKEY -AadTenantId $AAD_TENANT_ID -AzureSubscriptionId $AZURE_SUBSCRIPTION_ID -StorageAccountName "${AZURE_ACCOUNT_NAME}" -StorageAccountKey "${AZURE_ACCOUNT_KEY}" -StorageContainerNames "uploads,reports,tracking" -AzureCliVersion $AZURE_CLI_VERSION -AzureStorageConnectionString "${storage_connection_string}"
+powershell -file $INSTALLER_BASEPATH/Create-StorageContainer.ps1 \
```

Comment:
- [ ] This implementation implicitly assumes everyone using oxa-tools need to have their uploads container public. Is that always correct? If not, additional plumbing is needed to make this configurable. If yes, then this is moot.